projects/plugins/protect/src/js/routes/scan/index.jsx

```diff
@@ -39,11 +39,7 @@ const ScanPage = () => {
 	}
 
 	const hasHistory = history && history.threats.length;
-<<<<<<< HEAD
 	const showResults = !! status.threats.length || hasHistory;
-=======
-	const showResults = status.threats.length || hasHistory;
->>>>>>> 1850b0c9a6 (Protect: only show Threats DataViews when scan is in progress or has results to show)
```

Comment:
Similar to the [base PR](https://github.com/Automattic/jetpack/pull/40621#discussion_r1887659093), and quite possibly needing [a change](https://github.com/Automattic/jetpack/pull/40620#discussion_r1887657340) to the `status.threats.length` logic.